src/components/Dialog/CalendarDialog.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-dialog
+    :model-value="props.modelValue"
+    @update:model-value="props.modelValue"
+    transition="dialog-top-transition"
+    width="auto"
+  >
+    <v-card>
+      <v-date-picker
+        v-model="datePicker"
+        color="green lighten-1"
+        @update:model-value="dateHandler"
+        @click:cancel="dateHandler"
```

Comment:
Should we also have e `@click:ok`? Or it doesn't work like that?

Reply:
there is @click.save function, but it updates the date on the 2nd save. On the first save, it alaways takes today's date for some reason, while         @update:model-value="closeDialog" is working properly as intended